source/css/sass/mixins/_spacing.scss

```diff
@@ -102,26 +102,26 @@
 }
 
 @mixin h2-spacing() {
-  @include block-spacing();
+  @include block-spacing($end: $baselinegrid-space-small * 0.875);
```

Comment:
0.875 seems magic. Could put into an appropriately-named sass variable for improved readability.

Reply:
Indeed it is, but as far as I can tell it really is a magic number! Not sure it has a good name as a result...